app/assets/javascripts/backbone/views/project_step_modal_view.coffee

```diff
@@ -21,7 +22,9 @@ class MS.Views.ProjectStepModalView extends Backbone.View
     @done = done
     date = options.date || ''
     parentId = options.parentId if options.parentId
-    @loadContent("/admin/project_steps/new?loan_id=#{loanId}&context=timeline_table&date=#{date}&parent_id=#{parentId}")
+    precedingStepId = options.precedingStepId if options.precedingStepId
```

Comment:
Please change to `precedentId`. This is what I want to use throughout the code to refer to these things. We can do Preceding Step in the UI, but in code I prefer precedent b/c it's shorter.